tests/testApiOperations.php

```diff
@@ -0,0 +1,173 @@
+<?php
+/**
+ * Contains Class TestApiOperations.
+ *
+ * @package WP-Auth0
+ *
+ * @since 3.8.1
+ */
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Class TestApiOperations.
+ */
+class TestApiOperations extends TestCase {
+
+	use HttpHelpers;
+
+	use RedirectHelpers;
+
+	use SetUpTestDb;
+
+	/**
+	 * Instance of WP_Auth0_Options.
+	 *
+	 * @var WP_Auth0_Options
+	 */
+	public static $opts;
+
+	/**
+	 * WP_Auth0_ErrorLog instance.
+	 *
+	 * @var WP_Auth0_ErrorLog
+	 */
+	protected static $error_log;
+
+	/**
+	 * Setup for entire test class.
+	 */
+	public static function setUpBeforeClass() {
+		parent::setUpBeforeClass();
+		self::$opts      = WP_Auth0_Options::Instance();
+		self::$error_log = new WP_Auth0_ErrorLog();
+	}
+
+	/**
+	 * Test that a basic create connection command requests properly.
+	 */
+	public function testThatCreateConnectionRequestsCorrectly() {
```

Comment:
Odd naming. Consider the suffix you've used above "WithoutErrors". Applies to more tests on this file